src/sentry/tasks/store.py

```diff
@@ -344,6 +344,32 @@ def symbolicate_event(cache_key, start_time=None, event_id=None, **kwargs):
     )
 
 
+@instrumented_task(
+    name="sentry.tasks.store.symbolicate_event_low_priority",
+    queue="events.symbolicate_event_low_priority",
+    time_limit=settings.SYMBOLICATOR_PROCESS_EVENT_HARD_TIMEOUT + 30,
+    soft_time_limit=settings.SYMBOLICATOR_PROCESS_EVENT_HARD_TIMEOUT + 20,
+    acks_late=True,
+)
+def symbolicate_event_low_priority(cache_key, start_time=None, event_id=None, **kwargs):
```

Comment:
Is this task called somewhere, or will that be part of a later PR?

Reply:
Yes, that's going to happen in a later PR.